### PR TITLE
UI/task details page

### PR DESCRIPTION
**Pull request by ankitbajpai65**
**Description**
Fixed CSS of the task details page and made it responsive for mobile. 

**Issue**
https://github.com/Real-Dev-Squad/website-status/issues/685#issue-1787906964

### Images/video of the change:
image
image

https://github.com/Real-Dev-Squad/website-status/assets/85179394/23a017c7-ff85-4503-96f3-2e2b92bc0f6c

### CODE REVIEW

**Review by vinit717 — COMMENTED** [review]
Please run  lint-fix and prettier

**Comment by ankitbajpai65** [comment]
Hey @vinit717, I made the above mentioned changes. You can now check the changes

File: package.json [hunk 1]
```diff
@@ -7,13 +7,12 @@
...
     "run-command:reverse-ssl": "local-ssl-proxy --source 443 --target 3000",
     "dev": "run-p run-command:*",
-    "build:staging":"cp .env.development .env.local && next build",
+    "build:staging": "cp .env.development .env.local && next build",
     "build": "next build",
     "start": "next start -p $PORT",
     "test": "jest --coverage",
     "test-watch": "jest --watch ",
     "lint": "next lint",
-    "lint-fix": "eslint . --fix --ext js,jsx,ts,tsx",
```
Comment (vinit717) [inline package.json:16]: Why you remove this ?

File: src/components/Layout/Layout.module.scss [hunk 2]
```diff
@@ -10,6 +10,12 @@
     margin: 0 2em;
 }
 
+@media (max-width: 450px) {
+    .wrapper {
+        margin: 0 10px;
+    }
+}
+
```
Comment (vinit717) [inline src/components/Layout/Layout.module.scss:18]: Put media query always at the bottom of the file

File: src/components/taskDetails/index.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
 
     const { data: userData, isUserAuthorized } = useUserData();
 
-    const [isEditing, setIsEditing] = useState<boolean>(false);
+    const [isEditing, setIsEditing] = useState<boolean>(true);
```
Comment (vinit717) [inline src/components/taskDetails/index.tsx:61]: Why you changed this

**Review by vinit717 — CHANGES_REQUESTED** [review]

**Comment by ankitbajpai65** [comment]
Hey @vinit717, can you check again. Thanks

File: package.json
Comment (vinit717) [inline package.json:1]: Why there is a need to change package json it is just a simple CSS change

File: src/components/navBar/navBar.module.scss [hunk 4]
```diff
@@ -82,28 +83,33 @@ $theme-active: #87d870;
...
         text-underline-offset: $offset;
     }
 }
+
 @media (max-width: 970px) {
     .navBar {
         display: flex;
         flex-direction: row;
         flex-wrap: wrap;
         position: relative;
```
Comment (vinit717) [inline src/components/navBar/navBar.module.scss:111]: Please place this media query at the bottom of the file

**Review by vinit717 — CHANGES_REQUESTED** [review]

File: src/components/navBar/navBar.module.scss (hunk 4 above)
Comment (ankitbajpai65) [reply src/components/navBar/navBar.module.scss:111]: It is at the bottom, please check

**Comment by ankitbajpai65** [comment]
@vinit717, Please check again. Thanks.

**Review by vinit717 — COMMENTED** [review]
You have to add only CSS file changes until there is no Class name is added

**Comment by ankitbajpai65** [comment]
This pull request work is done in this https://github.com/Real-Dev-Squad/website-status/pull/779.
Please have a look into this. Thanks.

**Pull request closed without merging**